src/python/pants/backend/awslambda/python/register.py

```diff
@@ -6,10 +6,10 @@
 See https://www.pantsbuild.org/docs/awslambda-python.
 """
 
-from pants.backend.awslambda.python import lambdex
```

Comment:
Sorry for roping in this change. The lockfile generation didn't like having two backends containing a subsytem, so I had to make the change so it wasn't in two places.

Reply:
This is a good change! We don't want to define the subsystem twice.

Reply:
If you want, you could land this is a prework PR. I don't think that's necessary because this change is fairly minor, but feel free to so you separate refactor vs. new feature.